vmdk_plugin/utils/plugin_server/plugin_server_linux.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package plugin_server
```

Comment:
Sorry, wanted to rephrase the comment and missed. Basically, is it possible to ensure that only the relevant code is even compiled into the plugins. Like if all plugin servers are part of the plugin_server package does all of the code get included into the binary? Can that be avoided?